.depcheckrc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "ignores": [
+    "@lavamoat/allow-scripts",
+    "@lavamoat/preinstall-always-fail",
+    "@metamask/auto-changelog",
+    "@types/*",
+    "prettier-plugin-packagejson",
+    "ts-node",
+    "@storybook/addon-actions",
+    "@storybook/react-webpack5",
+    "babel-loader",
+    "rimraf",
+    "sass",
+    "webpack",
+    "@yarnpkg/core",
+    "@yarnpkg/cli",
+    "clipanion",
+    "@yarnpkg/fslib"
```

Comment:
Hey @mcmire I'm not sure why buy when I run `yarn lint` it picks up that the yarnpkg are missing dependencies? 

```
    "@yarnpkg/core",
    "@yarnpkg/cli",
    "clipanion",
    "@yarnpkg/fslib"
```

Reply:
This makes sense. All of these dependencies are used by Yarn, and `depcheck` is unable to see that these are used, since they're never imported anywhere. So this is the right call.